src/modules/Dashboard/components/ImagesCardView/ImageCard/ImageCard.tsx

```diff
@@ -48,8 +50,21 @@ export default class ImageCard extends Component<IProps, IState> {
 		image: {
 			thumb: '',
 		},
+		textFilter: '',
 	};
 
+	public findInTextWrapper = (text: string) => {
+		if (this.props.textFilter) {
```

Comment:
multiple usage of props

Reply:
i think you must use that inside selectors. here must be just an action creator, you type something, selector for items recalculates and return a new array of items.

Reply:
i think it is critical. @CodeAnnihilator ?

Reply:
right

Reply:
split this whole file on smaller logical components